_posts/2022-02-01-Midi-Mixerboard-Control.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "Control the Jamulus Mixerboard using MIDI"
+lang: "en"
+author: "henkdegroot"
+heading: "How to control the Jamulus Mixerboard using a MIDI Control Surface in Windows using JACK"
+---
+
+This guide explains how to setup your system in order to control the Jamulus Mixerboard using a MIDI Control Surface device (or similar) using JACK.
+The process will be similar for Linux (this also requires JACK and a2jmidid), while for macOS you can use MIDI Studio.
+
```

Comment:
Add a "More" html comment here. See other posts.

Reply:
What "More" html....I really can't find any more.

Reply:
This tag:

`<!--more--> `